CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Add `CheckInUtils.withCheckIn` which abstracts away some of the manual check-ins complexity ([#2959](https://github.com/getsentry/sentry-java/pull/2959))
 
+### Fixes
+
+- SentryOkHttpEvent report exceptions only on the call root span ([#2961](https://github.com/getsentry/sentry-java/pull/2961))
```

Comment:
It's actually HTTP errors we're catching here, how about naming it this way? 
```suggestion
- Always attach `SentryOkHttpEvent` errors to the call root span ([#2961](https://github.com/getsentry/sentry-java/pull/2961))
```

Reply:
Actually it's both
any exception caught by the listener is now associated to the call root span only, and the `SentryHttpClientException` is also associated to the call root span only.
What about "Always attach `OkHttp` errors and `Http Client Errors` only to call root span"?